utils/flags/flags.go

```diff
@@ -183,6 +183,24 @@ func GetCliFlags() []cli.Flag {
 			Usage:   "When using proxy backends, sets the number of Goroutines to process parallel uploads to backend.",
 			EnvVars: []string{"BAZEL_REMOTE_NUM_UPLOADERS"},
 		},
+		&cli.StringFlag{
+			Name:    "grpc_proxy.url",
+			Value:   "",
+			Usage:   "The base URL to use for a grpc proxy backend.",
+			EnvVars: []string{"BAZEL_REMOTE_GRPC_PROXY_URL"},
+		},
+		&cli.StringFlag{
+			Name:    "grpc_proxy.key_file",
+			Value:   "",
+			Usage:   "Path to the key used to autheticate with the proxy backend. Enables mTLS.",
```

Comment:
The `key_file` / `cert_file` help texts should each mention that using one requires the other.